store/player.js

```diff
@@ -98,19 +90,10 @@ export const mutations = {
 // Actions
 export const actions = {
   async [$actions.loadSettings] ({ dispatch, commit }) {
-    // Check if we have access to localStorage
-    if ( localStorage === null ) {
-      console.error( 'Cannot access localStorage' );
-      return;
-    }
-
-    let keeplive = null;
-    try {
-      keeplive  = localStorage.getItem( 'keepLive' );
-    } catch ( error ) {
-      console.error( `Failed to get 'keepLive' from localStorage`, error );
-    }
-
+    // TODO(diingus): It is not obvious to me why keepLive is read separately, as it's
+    //                never written to directly (only through saveToLocalStorage() which
+    //                saves it wrapped in a single object), and git grep is inconclusive
+    const keeplive = utils.readFromLocalStorage( 'keepLive' );
```

Comment:
Was this comment resolved after discord discussions?

Reply:
I thought I removed this :man_shrugging: